### ISSUE TITLE
Maintenance: Reduce size of docker imge

### ISSUE THREAD

**Issue by roschaefer** [Human-Connection/API#168]
Our current `edge` docker image is 205MB big. That's way too large. Probably there is some potential for minimization of the image size. https://hub.docker.com/r/humanconnection/api-feathers/tags/

**Comment by realB12** [comment]
My opinion is - and I know, you don't even listen - following a distributed Microservice oriented Architecture would have been a much better approach...
For instance I would make the database and API-Server two independent Containers that are both running as a)  cloud instances every developer can access  and b) local instances for developers working on a specific container (when you have to work on several container/services in parallel) there most probably is something wrong with your development process. 
But ok - I stop her - just giving this as a teaser what kind of discussion should be held here. 
Maybe you want to hear more. Or just forget about. I do not really care ..., I see you just running into issues, that might not have been an issue, when done differently right from the beginning.

**Comment by mimicc83** [comment]
@realB12 I think you need to start at the very basics of feathersjs here: https://feathersjs.com/ as we already have a very granular micro service architecture setup, just not spread into different projects which isn't necessary as of today. However, if you feel that our current setup isn't working well you're invited to bring any pitfalls to our concerns (with proof of code/use or showcase please) and we'll discuss it. You're also much welcome to send us a draft as @roschaefer mentioned. 

If you're new to nodejs, feathers and nuxt feel free to join our discord and we will help you to gather the required knowledge. If you already have some or more deep knowledge, even better! Just jump on the train, contribute code or ideas. For discussion on topic's please open a separate issue if there isn't one ongoing already. 

We also do pairing sessions every now and then or on demand and a weekly or bi-weekly developer meeting in voice/video chat will follow soonish. 

Thanks and have a nice day!

**Comment by Lulalaby** [comment]
Without getting too close, I think you're exaggerating.
We listen to you, but we have our way, which works very well.
I don't know you personally and so I can't tell if you're trolling or something.
"a fool with a tool is still a fool" is already quite offensive against the developers here who put their life and lifeblood into it.

Please hold back a bit.

**Comment by mimicc83** [comment]
then tell me please how you want to contribute to our software? And why exactly do we need to listen to you? Who are you? Linus Torvalds?
All you did as of now is making BS comments which have NO value, come onto discord, join the discussions on real problems, start contributing or roll your own, it's open source. 

Oh and when you are done with feathers I hope you notice that almost every aspect can be put into a microservice. And even if it should be obvious let me enlight you: feathers is not a tool 

Last time I clearly provide you with an anwser on this matter:

- WE decide which hosting to use 
- WE decide which technology stack we go with
- WE decide the architecture and every other matter of the software

WE refers to the team including all contributors together, participation takes place on related github issues and on discord (you are free to say hi in discord since you're already on the server), join the discussions and help us get stuff done.

**Comment by realB12** [comment]
No "WE" is the stuff you already have in place and the core team defending it as hell ;-)
It is ok for me. I am out.

**Comment by Lulalaby** [comment]
I'll say it only once now:
We (the team and several helpers) are all working hard on it.
Do something besides commenting, complaining and spurring on or just leave us alone.
